test/core/form_test.js

```diff
@@ -123,6 +123,18 @@ describe('<Form />', () => {
     expect(render.at(0).instance().value).to.eql(values);
   });
 
+  it('even if those values are falsy', () => {
+    const values = { username: null, password: '' };
+    const render = mount(
+      <Form defaultValue={values}>
+        <TextInput name="username" defaultValue="droo" />
+        <PasswordInput name="password" />
+      </Form>
+    );
+
+    expect(render.at(0).instance().value).to.eql(values);
```

Comment:
can you also verify here that the TextInput has the right value too? in case the form and the input go splitbrain

Reply:
also, how did it work? the username is `null` here, und your patch checks against `undefined`

Reply:
I wanted to specifically allow `null` as a falsy value that conveys meaning beyond simple booleans.